### PR TITLE
fix(getFeature): Add project uuid to response

### PR DESCRIPTION
**Pull request by gagantrivedi**
* Fix GetProjects 
* Add projectUUID to response of GetFeature

### CODE REVIEW

File: client_test.go [hunk 1]
```diff
@@ -479,8 +477,12 @@ func TestGetFeature(t *testing.T) {
 	// Given
 	masterAPIKey := "master_api_key"
 	featureUUID := "10421b1f-5f29-4da9-abe2-30f88c07c9e8"
+	projectUUID := "10421b1f-5f29-4da9-abe2-30f88c07c9e8"
```
Comment (matthewelwell) [inline client_test.go:480]: We should probably use different UUIDs for feature / project to avoid confusion

File: client_test.go [hunk 2]
```diff
@@ -479,8 +477,12 @@ func TestGetFeature(t *testing.T) {
...
+
+	mux := http.NewServeMux()
+
+	mux.HandleFunc("/api/v1/features/get-by-uuid/10421b1f-5f29-4da9-abe2-30f88c07c9e8/", func(rw http.ResponseWriter, req *http.Request) {
```
Comment (matthewelwell) [inline client_test.go:484]: Is there a reason we're not using the `featureUUID` variable here and just hardcoding the UUID again?

File: client_test.go [hunk 3]
```diff
@@ -479,8 +477,12 @@ func TestGetFeature(t *testing.T) {
...
 
-	server := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
 		assert.Equal(t, "/api/v1/features/get-by-uuid/10421b1f-5f29-4da9-abe2-30f88c07c9e8/", req.URL.Path)
```
Comment (matthewelwell) [inline client_test.go:486]: as above

File: client_test.go [hunk 4]
```diff
@@ -489,7 +491,15 @@ func TestGetFeature(t *testing.T) {
 		_, err := io.WriteString(rw, CreateFeatureResponseJson)
 		assert.NoError(t, err)
 
-	}))
+	})
+
+	mux.HandleFunc("/api/v1/projects/1/", func(rw http.ResponseWriter, req *http.Request) {
```
Comment (matthewelwell) [inline client_test.go:496]: Where does project id `1` come from?

File: client_test.go (hunk 1 above)
Comment (gagantrivedi) [reply client_test.go]: Yeah, good point

File: client_test.go (hunk 2 above)
Comment (gagantrivedi) [reply client_test.go]: Yeah, I have moved them to constants

File: client_test.go [hunk 5]
```diff
@@ -59,6 +59,8 @@ const UpdateFeatureStateResponseJson = `
   "change_request": null
 }
 `
+const FeatureID int64 = 1
```
Comment (gagantrivedi) [inline client_test.go:62]: Note: they are hardcode here because go does not support string interpolation for constants

Comment (matthewelwell) [reply client_test.go:62]: Should we add the projectUUID here too?

Comment (gagantrivedi) [reply client_test.go:62]: Isn't more relevant here: https://github.com/Flagsmith/flagsmith-go-api-client/pull/3/files#diff-4dc56e64f310cd8c12bb00f75c09e83f5d875563ea9901788e525487dc6788b3R175?

Comment (matthewelwell) [reply client_test.go:62]: I'm confused why we are still hardcoding it in so many places though?

Comment (gagantrivedi) [reply client_test.go:62]: well,  because go does not support string interpolation for constants

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**